resources/language/resource.language.fr_fr/strings.po

```diff
@@ -66,7 +66,7 @@ msgstr "Vérifiez vos identifiants"
 
 msgctxt "#30010"
 msgid "Lists of all kinds"
-msgstr "Listes de toutes sortes"
+msgstr "Tous les genres"
```

Comment:
I thought this translation was more explicit, especially because it comes right after `Tv Show genres` and `Movies genres`. Is my understanding correct: this section lists all the movies and TV shows by genres right?
 If you prefer I could use `Liste de tous les genres` but it sounds less "official" according to me.
What do you prefer?

Side note: I think the english string should be `Lists of all genres` no?

Reply:
the word "genre" for the name of this menu was not used because there are two type of different lists in netflix:
-the menus genres (like "Tv Show genres") actually list the real genres by type
-the menu "list of all kinds" is not a list of genres, but a sort of dynamic sub-lists that show sub-menus not constant over time

that's the reason for this difference, but if in your language "Tous les genres" means same thing of "Lists of all kinds" then it is ok

Reply:
Ok so I was wrong. I opened this menu but I overlooked the content and I thought it was a list with all the genres. I will revert my change, the previous translation was correct.